redis/config.go

```diff
@@ -0,0 +1,36 @@
+package redis
```

Comment:
I would suggest to make it a separate package, and put it inside subdirectory, something like redis/config/config.go